### PR TITLE
Fix https://github.com/Keats/tera/issues/513

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

File: src/parser/tera.pest [hunk 1]
```diff
@@ -190,10 +190,10 @@ super_tag        = !{ variable_start ~ "super()" ~ variable_end }
...
 text       = ${ (!(block_start) ~ ANY)+ }
 
-raw_tag    = { tag_start ~ "raw" ~ tag_end }
-endraw_tag = { tag_start ~ "endraw" ~ tag_end }
-raw_text   = { (!endraw_tag ~ ANY)* }
-raw        = !{ raw_tag ~ raw_text ~ endraw_tag }
+raw_tag    = !{ tag_start ~ "raw" ~ tag_end }
+endraw_tag = !{ tag_start ~ "endraw" ~ tag_end }
+raw_text   = ${ (!endraw_tag ~ ANY)* }
+raw        = ${ raw_tag ~ raw_text ~ endraw_tag }
```
Comment (jakeswenson) [inline src/parser/tera.pest:196]: I know i could go read the pest documentation, but could you maybe describe what this subtle change is doing?

File: src/renderer/tests/whitespace.rs [hunk 2]
```diff
@@ -17,7 +17,7 @@ fn can_remove_whitespace_basic() {
         ("  {%- if false -%}\n {{numbers}}\n {% elif false -%} Nope {% else %} else {%- endif -%} ", " else"),
         ("  {%- set var = 2 -%} {{var}}", "2"),
         ("  {% set var = 2 -%} {{var}}", "  2"),
-        ("  {% raw -%}{{2}} {% endraw -%} ", "  {{2}}"),
+        ("  {% raw -%}{{2}} {% endraw -%} ", "  {{2}} "),
```
Comment (jakeswenson) [inline src/renderer/tests/whitespace.rs:20]: Wouldn't it be better to have this:
```suggestion
        (" {% raw -%} {{2}} {% endraw -%} ", " {{2}} "),
```

**Comment by jakeswenson** [comment]
by the way i think you meant to link to #513

File: src/parser/tera.pest (hunk 1 above)
Comment (Keats) [reply src/parser/tera.pest:196]: It just changes the way pest handle whitespace in those expression, before they were ignored and there we just enable it again.

**Pull request merged**